assets/app/vue/components/SignUpFlowForm.vue

```diff
@@ -66,4 +78,10 @@ const onSubmit = () => {
 :deep(.tbpro-input) {
   box-sizing: border-box;
 }
+
+.button-container {
+  display: flex;
+  flex-direction: row;
+  gap: 0.25rem;
```

Comment:
```suggestion
  gap: 1rem;
```

The buttons are actually quite shy. We should make sure they're comfortable by giving them a bit more space. (And also so the user doesn't misclick.)